pkg/netbox/api/prefix_claim.go

```diff
@@ -53,16 +53,31 @@ func (r *NetboxClient) RestoreExistingPrefixByHash(hash string) (*models.Prefix,
 	}, nil
 }
 
-func isRequestingTheEntireParentPrefix(prefixClaim *models.PrefixClaim) (bool, error) {
+func isRequestingInvalidPrefixLength(prefixClaim *models.PrefixClaim, prefixFamily int64) (bool, string, error) {
```

Comment:
Is there a specific reason to do `(bool, string, error)` instead of just `(err)`?

As the only use case is 
```
if ret, message, err := isRequestingInvalidPrefixLength(prefixClaim,
	*responseParentPrefix.Payload.Results[0].Family.Value); err != nil {
	return nil, err
} else if ret {
	return nil, errors.New("invalid prefix length: " + message)
}
```
I am not sure if splitting out `ret` and then generating an error from the string using `errors.New` makes sense. :)

Reply:
It's to distinguish between the case A "validation fails" and case B "prefix length is invalid".
In case B the "isRequestingInvalidPrefixLength" the function is working as expected. 
I will change the function name and will return only an error.

Reply:
Since both `err != nil` and `ret == true` will throw an error to the caller, I think it makes sense that we just throw an error from `isRequestingInvalidPrefixLength` with all the associated error messages. 

Thanks for making the changes!